### ISSUE TITLE
Fix Python3 syntax errors in twisted/application/app.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7143]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7143|
|Type|enhancement|
|Created|2014-04-14 18:21:00Z|
|Branch|https://github.com/twisted/twisted/tree/app-py3-7143|

Attachments:

* [t7143.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/308/308550d8e8c38ac5218c89645ca303aeb4a8c876/5b4d91a71af653279923709957f3efbd3ae3d9ff.diff) (3259 bytes) - added by Alex on 2014-04-14 18:21:12Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7143 7143
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_app_py3_7143 branches/app-py3-7143
branch_author__radix radix
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1397499660000000 1397499660000000
changetime__1474198049187875 1474198049187875
version__None None
owner__Alex Alex

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix commented|
|-|-|

(In [42215]) Branching to app-py3-7143.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix commented|
|-|-|

You changed a long line without fixing it to be <80 columns. If you fix those errors I think it's mergeable.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to Alex|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|Alex commented|
|-|-|

Radix applied my patch fixing this.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_6"></a>|@radix set status to `closed`|
|-|-|

(In [42245]) Merge app-py3-7143: Update app.py syntax to py3

Author: Alex
Reviewer: radix
Fixes: [#7143](https://github.com/twisted/twisted/issues/7143)

Update twisted/application/app.py to python3-compatible syntax.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
